src/com/google/javascript/rhino/Node.java

```diff
@@ -3045,6 +3045,10 @@ public boolean isTemplateLitSub(){
   }
 
   public boolean isThis() {
```

Comment:
Then let me make a `NodeUtils.isThisOrAlias(Node n)` type method and look through the code base for any reference to `n.isThis()` type calls and switch it where needed.